challengers/Joyal-Poudel/day4-fastapi-websocket-chat/main.py

```
@@ -0,0 +1,407 @@
+        }
+
+    def disconnect(self, websocket: WebSocket):
+        if websocket in self.active_connections:
+            self.active_connections.remove(websocket)
+            del self.users[id(websocket)]
+
+    async def broadcast(self, data: dict):
+        for connection in self.active_connections:
+            try:
+                await connection.send_json(data)
+            except:
```

Ruff (0.15.14)

[error] Do not use bare `except`

(E722)